modules/nf-core/vgan/haplocart/meta.yml

```
@@ -0,0 +1,46 @@
+name: "vgan_haplocart"
+description: Human mtDNA haplogroup classification
+keywords:
+  - mtDNA
+  - mitochondria
+  - pangenome
+  - haplogroup
+  - variation graph
+tools:
+  - "vgan":
+      description: "Suite of tools for pangenomics built using vg"
+      homepage: "https://github.com/grenaud/vgan"
+      documentation: "https://github.com/grenaud/vgan"
+      tool_dev_url: "https://github.com/grenaud/vgan"
+      doi: ""
+      licence: "['GPL v3']"
+
+input:
+  - INPUT1:
+      type: file
+      description: Input file
+      pattern: "*.{fq,fq.gz,fastq,fastq.gz}"
+
+  - INPUT2:
+      type: file
+      description: Optional second FASTQ file for non-interleaved paired-end reads
+      pattern: "*.{fq,fq.gz,fastq,fastq.gz}"
+
```

```suggestion
  - reads:
      type: file
      description: Input file
      pattern: "*.{fq,fq.gz,fastq,fastq.gz}"
  - reads2:
      type: file
      description: Optional second FASTQ file for non-interleaved paired-end reads
      pattern: "*.{fq,fq.gz,fastq,fastq.gz}"

```

Can you also add `meta` (this you can copy paste from other modules) and the `hf-files` to this file?

---

Yes, agree, mising `hc_files`

---

I have made the changes.